### PR TITLE
Add Carousel thumbnails/responsive height and Masonry lightbox

### PR DESCRIPTION
**Pull request by grabovacnem**

### CODE REVIEW

**Comment by richtabor** [comment]
@grabovacnem Could you resolve these conflicts? It looks like the recent refactoring caused some regressions. The save functions are now in the save.js files within the block.

File: dist/images/masonry-modal/arrow-left.svg [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 19.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```
Comment (jonathanbardo) [inline dist/images/masonry-modal/arrow-left.svg]: Is it possible to remove this?

File: dist/images/masonry-modal/arrow-left.svg [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 477.175 477.175" style="enable-background:new 0 0 477.175 477.175;" xml:space="preserve">
+<g>
+	<path d="M145.188,238.575l215.5-215.5c5.3-5.3,5.3-13.8,0-19.1s-13.8-5.3-19.1,0l-225.1,225.1c-5.3,5.3-5.3,13.8,0,19.1l225.1,225
+		c2.6,2.6,6.1,4,9.5,4s6.9-1.3,9.5-4c5.3-5.3,5.3-13.8,0-19.1L145.188,238.575z"/>
+</g>
```
Comment (jonathanbardo) [inline dist/images/masonry-modal/arrow-left.svg]: There are many empty groups here that are not needed. If possible, please remove

File: dist/images/masonry-modal/close.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M23 20.168l-8.185-8.187 8.185-8.174-2.832-2.807-8.182 8.179-8.176-8.179-2.81 2.81 8.186 8.196-8.186 8.184 2.81 2.81 8.203-8.192 8.18 8.192z"/></svg>
```
Comment (jonathanbardo) [inline dist/images/masonry-modal/close.svg]: Let's keep the formatting consistent on svgs and include the xml declaration and have the svg on multiple line like the arrow-right file.

File: dist/images/masonry-modal/arrow-left.svg (hunk 1 above)
Comment (richtabor) [reply dist/images/masonry-modal/arrow-left.svg]: Yea, this SVG needs to be cleaned.

Comment (jrtashjian) [reply dist/images/masonry-modal/arrow-left.svg]: This is what I've used to clean up SVGs: https://jakearchibald.github.io/svgomg/

**Pull request closed without merging**